src/components/auth/login/WelcomeScreen.js

```diff
@@ -32,16 +32,20 @@ import {
 
 import { withStyles } from '../../../lib/styles'
 import Illustration from '../../../assets/Auth/torusIllustration.svg'
+import { GlobalTogglesContext } from '../../../lib/contexts/togglesContext'
 
 const AuthScreen = Config.torusEnabled ? AuthTorus : Auth
 
 const WelcomeScreen = ({ theme, styles, screenProps, navigation }) => {
   const { navigate } = navigation
+  const { hasSyncedCodePush } = useContext(GlobalTogglesContext)
 
   const onGetStarted = useCallback(() => {
-    fireEvent(CLICK_GETSTARTED)
-    navigate('Auth')
-  }, [navigate])
+    if (!isMobileNative || (isMobileNative && hasSyncedCodePush)) {
```

Comment:
```suggestion
    if (!isMobileNative || hasSyncedCodePush) {
```

as I suppose `hasSyncedCodePush` never would be `true` at web ?

Reply:
ye that is true. fixed